<?xml version="1.0"?>

<launch>


  <node name="path_recorder" pkg="path_recorder" type="path_recorder" output="screen">      
        
      <!-- <remap from="/fix" to="/carla/ego_vehicle/gnss/gnss1/fix"/> -->
      
       <!-- options :  true = gnss , false = local_pose,   -->
      <param name="path_record_with_gnss" value="True"/> 
      <!-- when local_pose is selected,  origin_lat, origin_lon, origin_att =>> x, y, z  -->
      <param name="map_origin_lat" value="35.575307085"/> 
      <param name="map_origin_lon" value="129.190738787"/>  
      <param name="map_origin_att" value="36.918"/> 

      <param name="osm_file_name" value="$(find path_recorder)/maps/carla_test.osm"/> 
      <param name="visualize_path" value="True"/> 
      <param name="line_resolution" value="10.0"/> 
      <param name="point_resolution" value="1.0"/> 

      

      
  </node>


  
</launch>
